templates/deployments/bitcoind-chain-coordinator.template.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: bitcoind-chain-coordinator
+    app.kubernetes.io/instance: "{user_id}"
+    app.kubernetes.io/managed-by: stacks-devnet-api
+    app.kubernetes.io/name: bitcoind-chain-coordinator
+  name: bitcoind-chain-coordinator
+  namespace: "{namespace}"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: bitcoind-chain-coordinator
+      app.kubernetes.io/instance: "{user_id}"
+      app.kubernetes.io/managed-by: stacks-devnet-api
+      app.kubernetes.io/name: bitcoind-chain-coordinator
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: bitcoind-chain-coordinator
+        app.kubernetes.io/instance: "{user_id}"
+        app.kubernetes.io/managed-by: stacks-devnet-api
+        app.kubernetes.io/name: bitcoind-chain-coordinator
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: cloud.google.com/gke-preemptible
+                    operator: DoesNotExist
+      containers:
+        - command:
+            - /usr/local/bin/bitcoind
+            - -conf=/etc/bitcoin/bitcoin.conf
+            - -nodebuglogfile
+            - -pid=/run/bitcoind.pid
+          image: quay.io/hirosystems/bitcoind:devnet-v3
```

Comment:
We might want to template this image, as well as any other image being used in other containers. Otherwise if we want/need to change the images being used, it'll require a new release. It's much easier if we're able to control this aspect through infra config, like an ENV variable we set in the devnet-api itself.

Reply:
I've created an issue for this: #79